schemas/hub/tags.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$id": "https://raw.githubusercontent.com/guardrails-ai/interfaces/schemas/hub/tags.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Tags",
+  "type": "object",
+  "properties": {
+    "language": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "en",
+          "all"
+        ]
+      }
+    },
+    "certification": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "Guardrails Certified"
+        ]
+      }
+    },
+    "contentType": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "string",
+          "object",
+          "list",
+          "integer",
+          "float",
+          "sql",
+          "code",
+          "csv",
+          "python"
+        ]
+      }
+    },
+    "infrastructureRequirements": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "ML",
+          "NA",
+          "LLM",
+          "rule"
```

Comment:
Rule?

Reply:
All of the tag values are taken from what is actually in the manifests in [the hub](https://github.com/guardrails-ai/guardrails-hub).  If there's a value we don't like we can remove it, this was just a starting point.

Reply:
oh I meant "Rule" instead of "rule" just to match